3570/3/rating_places/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'base_controller'
+
+class UsersController < BaseController
+  get '/users/new' do
```

Comment:
What kind of user I will see on this page?

Reply:
There is no any special kind of user on this page. This action renders a form for creating a new user

Reply:
If so you must send POST for create user to `/users`